include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -366,6 +366,12 @@ namespace LinearAlgebra
       bool
       has_ghost_elements() const;
 
+      /**
+       * Change the ghost state of this vector to @p ghosted.
+       */
+      void
+      set_ghost_state(const bool ghosted) const;
```

Comment:
I don't understand how this is supposed to work. Can you document what it means? I thought whether or not a vector is ghosted depends on which elements out of the range `[0,N)` it stores, and how that relates to the elements stored by other processes.